index.html

```diff
@@ -20,13 +23,166 @@
+                        <img id="ic_arrowID" src="ic_arrow_right.svg" alt="">
+                    </nav>
+                    
+                </div>
 
+            </div>
+            <div class="subGroup">
+                <img id="Rectangle21IG" src="Rectangle 21 (5).svg" alt="">
+                <div class="groupInfo">
+                    <h2 id="projectprojectNameName">Project name goes here</h2>
+                    <nav class="htmlBtn">
+                        <button id="htmlBtnID">HTML/CSS</button>
+                        <button id="htmlBtnID">Ruby on Rails</button>
+                        <button id="htmlBtnID">JavaScript</button>
+                    </nav>
+                    <nav id="seeProjectGrp">
+                        <button id="seeProjectBtn">See this project</button>
+                        <img id="ic_arrowID" src="ic_arrow_right.svg" alt="">
+                    </nav>
+                </div>
+            </div>   
+        </nav>
+    </section>
+    <section class="About_myself">
+        <nav class="group1">
+            <h2 id="titleh2">About</h2>
+            <p id="text">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <nav class="groupSetMyCV">
+                <button id="getMyCVBtn">Get my CV</button>
+                <img src="ic_download.svg" alt="">
+            </nav>
+        </nav>
+        <nav class="langagueGroup">
+            <nav>
+                <h3 id="titleh3">Languages</h3>
+                <ul class="list" id="listLanguages">
+                    <li>JavaScript (ES5/ES6)</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </nav>
+            <img id="ic_languages_mob" src="ic_languages_mob.svg" alt="" width="102px" height="102px">
+        </nav>
+        <nav class="FrameworksGroup">
+            <nav>
+                <h3 id="titleh3">Frameworks</h3>
+                <ul class="list" id="listFrameworks">
+                    <li>React.js</li>
+                    <li>Ruby  on rails</li>
+                    <li>RSpec</li>
+                    <li>Capibara</li>
+                </ul>
+            </nav>
+            <img id="ic_framework" src="ic_framework.svg" alt="" width="96px" height="96px">
+
+        </nav>
+        <nav class="SkillsGroup">
+            <nav>
+                <h3  id="titleh3">Skills</h3>
+                <ul class="list" id="listSkills">
+                    <li>Database Management</li>
+                    <li>CLI</li>
+                    <li>API Design</li>
+                    <li>Version Control</li>
+                    <li>Web Development</li>
+                </ul>
+            </nav>
+            <img id="ic_skills" src="ic_skills.svg" alt="" width="102px" height="102px">
```

Comment:
- [ ] **Good job so far 👏 👏** However, It would be better to use a multilevel list instead of using separated lists. as a multilevel list can also improve the accessibility of your content by providing a clear and structured outline of the information. 🙏 

Example:
```
<ul class="modals">
  <li>
    <span>Languages</span>
    <i class="fa-solid fa-arrow-down"></i>
    <ul>
      <li>
        <img src="images/javascript.png" alt="" />
        <p>JavaScript</p>
      </li>
      <li>
        <img src="images/html.png" alt="" />
        <p>HTML</p>
      </li>
      <li>
        <img src="images/css.png" alt="" />
        <p>CSS</p>
      </li>
    </ul>
  </li>
  
  <li>
    <span>Frameworks</span>
    <i class="fa-solid fa-arrow-right"></i>
  </li>

  <li>
    <span>Skills</span>
    <i class="fa-solid fa-arrow-right"></i>
  </li>
</ul>